collector.go

```diff
@@ -0,0 +1,94 @@
+package redigosrv
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+//RedigoCollector struct to access metrics
+type RedigoCollector struct {
+	publishTrafficSize prometheus.Counter
+	methodCalls        *prometheus.CounterVec
+	subscribeAmount    prometheus.Gauge
+	subscribeSuccesses prometheus.Counter
+	subscribeFailures  prometheus.Counter
+}
+
+// RedigoCollectorOptions struct to add custom name in metrics
+type RedigoCollectorOptions struct {
+	Separator string
+	Prefix    []string
+}
+
+const (
+	//PublishMetricMethodName has the name of the specific method
+	PublishMetricMethodName string = "Publish"
+	//SubscribeMetricMethodName has the name of the specific method
+	SubscribeMetricMethodName string = "Subscribe"
+)
+
+var redigoMetricLabels = []string{"method"}
+
+//RedigoCollectorDefaultOptions will return the instance of RedigoCollectorDefaultOptions with values default
+func RedigoCollectorDefaultOptions() RedigoCollectorOptions {
+	return RedigoCollectorOptions{
+		Separator: "_",
+		Prefix:    []string{"redigo"},
```

Comment:
Add the prefix as a `string` instead of an `[]string`.

Reply:
Done! :grin: